octopus-agent/src/main/java/octopus/teamcity/agent/OctopusMetadataBuildProcess.java

```diff
@@ -91,7 +92,19 @@ protected OctopusCommandBuilder createCommand() {
                 final String packageId = parameters.get(constants.getPackageIdKey());
                 final String packageVersion = parameters.get(constants.getPackageVersionKey());
 
-                final boolean forcePush = Boolean.parseBoolean(parameters.get(constants.getForcePushKey()));
+                final String forcePush = parameters.get(constants.getForcePushKey());
```

Comment:
Would it make sense to rename this variable now that it can be one of 3 behaviors?

Reply:
This is the migration issue again, until we work out how to do that we'll break current steps if we try to change the name or values